bftengine/src/bcstatetransfer/BCStateTran.cpp

```diff
@@ -1106,6 +1107,7 @@ bool BCStateTran::onMessage(const CheckpointSummaryMsg *m, uint32_t msgLen, uint
   LOG_DEBUG(getLogger(), "");
 
   FetchingState fs = getFetchingState();
+  LOG_DEBUG(getLogger(), "Fetching state is " << stateName(fs));
```

Comment:
This line is redundant, we have an assert one line after, so the fetching state is obvious.
Also, we don't need to write fetch state all the time.
now we a new log which prints every time state change (info level):
`FetchingState changed from`